jpo-asn-j2735-2024/src/main/java/us/dot/its/jpo/asn/j2735/r2024/RoadWeatherMessage/NTCIPEssPavementTreatmentLocation.java

```diff
@@ -34,6 +34,6 @@ public NTCIPEssPavementTreatmentLocation() {
   @JsonCreator
   public NTCIPEssPavementTreatmentLocation(String value) {
     this();
-    this.value = value;
+    this.setValue(value);

```

Comment:
```suggestion
    this.value = value;
```
issue (blocking): This type is an IA5String, not an Asn1Integer, let's keep it as is